np.sh

```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env sh
+
+if test -n "$(git status -uno --porcelain)"; then
+	echo "Unclean working tree. Commit or stash changes first." >&2;
+	exit 128;
```

Comment:
why exit 128? It seems it's "reserved" to invalid argument to exit http://www.tldp.org/LDP/abs/html/exitcodes.html